R/build-reference-index.R

```diff
@@ -48,7 +48,10 @@ data_reference_index_section <- function(section, pkg, depth = 1L) {
 
   contents <- tibble::tibble(
     path = section_topics$file_out,
-    aliases = purrr::map2(section_topics$funs, section_topics$name, ~ .x %||% .y),
+    aliases = purrr::map2(
+      section_topics$funs,
+      section_topics$name,
+      ~ if (length(.x) > 0) .x else .y),
```

Comment:
OK all done.

It's not 100% obvious to me it closes the other issues. Pretty sure it closes #395 but less sure about #374. I guess we'll find out!